tests/fragments.rs

```diff
@@ -1,13 +1,14 @@
 mod common;
 
+use crate::common::{ClientStub, ServerMoq};
 use laminar::{Config, DeliveryMethod};
+#[allow(unused_imports)]
+use log::{debug, error, info};
```

Comment:
removing? Probably done because of this WIP right?.

Reply:
Yes